pilot/pkg/proxy/envoy/v1/mock/discovery.go

```diff
@@ -115,6 +116,7 @@ func MakeService(hostname, address string) *model.Service {
 }
 
 // MakeExternalHTTPService creates mock external service
+// TODO: remove?
```

Comment:
no

Reply:
nvm, this is k8s external services --not v1alpha3 external services. it should stay.